training/prepare_dataset.py

```diff
@@ -107,122 +156,13 @@ def get_args() -> Dict[str, Any]:
         default="fp32",
         help="Data type to use when generating latents and prompt embeddings.",
     )
+    parser.add_argument("--seed", type=int, default=42, help="Seed for reproducibility.")
+    parser.add_argument(
+        "--num_artifact_workers", type=int, default=4, help="Number of worker threads for serializing artifacts."
+    )
     return parser.parse_args()
 
 
-def load_dataset_from_local_path(
```

Comment:
Where is this going?

Reply:
Do not require any more. Savings is done in `serialize_artifacts` that takes care of everything optionally

Reply:
But why mix the loading utilities with serialization? That feels unintuitive no?

Reply:
We aren't mixing anything, no? Instead of having these methods, we now load the video dataset by re-using our `VideoDatasetWithResizing` and `VideoDatasetWithResizeAndRectangleCrop`. Once the computation of latents and embeddings are done, we serialize with `serialize_artifacts`. Not sure I understand what you mean...

Reply:
My bad. Clear now.